### ISSUE TITLE
Cannot reproduce training results

### ISSUE THREAD

**Issue by kentaroy47** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#105]
Thanks for the great repo! It helped me alot to understand open-pose.

I have tried training with train_vgg.py, to train the model from scratch and the loss seemed to have converged.
However, I got weird inference results when testing.

- are the default lr and optimizer settings in the scripts correct?
- can you provide the training settings if anyone have successed in training from scratch?
